src/Synthesize.ml

```diff
@@ -254,7 +264,7 @@ let synthesize_lemmas
   in let input_examples = if String.equal synthesizer "myth" then ml_examples else coq_examples
   in Log.debug (Consts.fmt "Synth term is %s\n" (Sexp.string_of_sexpr curr_synth_term));
   let all_vars = List.append p_ctxt.vars conjecture.lfind_vars
-  in let coq_output_examples, ml_output_examples = (Evaluate.evaluate_coq_expr curr_synth_term input_examples p_ctxt all_vars conjecture.sigma (Some conjecture))
+  in let coq_output_examples, ml_output_examples = (Evaluate.evaluate_coq_expr curr_synth_term coq_examples p_ctxt all_vars conjecture.sigma (Some conjecture))
```

Comment:
Shouldnt this be input_examples since we do this `let input_examples = if String.equal synthesizer "myth" then ml_examples else coq_examples` earlier?

Reply:
Awesome! thanks for this great catch!